linter/examples/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash`

Reply:
Thank you @sthaha for the comment but I guess this needs to be changes throughout catalog :sweat_smile: because I took it from existing tekton tasks :sweat_smile: